packages/components/src/TransactionButton.tsx

```diff
@@ -72,6 +73,37 @@ export class TransactionButtonNoModal extends React.Component<TransactionButtonP
   }
 
   public render(): JSX.Element {
+    if (this.props.style && this.props.style === "inverted") {
```

Comment:
should this be a switch? or assign a variable like `Button` to the button component you want.

```
let Button: StyledComponentClass;
if (whatever condition) {
  Button = DarkButton;
}
return <Buton {...props}/>;
```